setup/odo-addbuilder.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+###################################################################################
+# Copyright (c) 2019 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v20.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###################################################################################
+
+# Import OpenShift image stream
```

Comment:
One question, wonder if user already had this image stream in the cluster, what would happen?
1) If it simply overwrite it without errors then I think we are good
2) If it throw some errors and block the importing, then we should check the image stream existence to avoid the errors

Reply:
@jingfuwang so tried that already.. basically since we're importing the image under the `codewind-odo-openjdk18` name it works even if the same image is already imported under another name.  Figured it's best to use a "codewind" specific name to avoid issues like this.  In the docs we can mention that it is important to ensure that this name is used for the image because that's what our tools look for when presenting the template options (the user could if they wanted to import another jdk stream, as long as that name is used)

Reply:
Yeah, we should specific image `codewind-odo-openjdk18` to avoid affecting other image streams in user's cluster. But my point is for reconfigure image stream with `exactly the same image stream name` (not just same image stream), use case is user may run the script again during codewind migration, what happen?

Reply:
Right so if script is run again, this error is shown: `error: --overwrite is false but found the following declared annotation(s): 'tags' already has a value (builder)` , but the existing image stream and its name are still there and continue to work. Now I suppose I could pass the `--overwrite=true` flag maybe?

Reply:
Yes, I think we can pass `--overwrite=true` so that user can update the image stream (the latest image stream may improve the performance) instead of seeing the errors above. Can you give a try the flag to see if the behaviour is expected?

Reply:
Yup that works as expected and I've made the change